packages/foreman/tfm/tfm.spec

```diff
@@ -79,11 +79,11 @@ Obsoletes: %{scl_prefix}rubygem-foreman_param_lookup < 0.3.0-3
 Obsoletes: %{scl_prefix}rubygem-foreman_reserve < 0.1.8.3-6
 Obsoletes: %{scl_prefix}rubygem-foreman_snapshot < 0.1.0-5
 Obsoletes: %{scl_prefix}rubygem-foremancli < 1.0-10
+Obsoletes: %{scl_prefix}rubygem-jquery_pwstrength_bootstrap_4 < 1.2.2-5
```

Comment:
What's the reason to move this around?

Reply:
None -- it was not present when I did this branch originally and the rebase moved it. I'll put it back